api/models.py

```diff
@@ -24,6 +24,11 @@
     FileUrl,
     SecretStr,
 )
+from beanie import Indexed, Document
+from fastapi_users.db import BeanieBaseUser
+from fastapi_users import schemas
+from beanie import PydanticObjectId, Indexed
+from typing import Optional
```

Comment:
I wonder if these should rather be in a separate module to avoid "polluting" the whole `api.models` namespace with `fastapi_users` imports.

Reply:
Yes, we should have a separate model file just like we have for pagination related models.